### PR TITLE
[Dictionary] Update ask file

### PR DESCRIPTION
**Pull request by livecodealice**
Moved cross platform note from Description to replace the broken note with the same content under types.
Fixed broken links.

### CODE REVIEW

File: docs/dictionary/command/ask-file.lcdoc [hunk 1]
```diff
@@ -42,13 +42,15 @@ suggested file name is provided in the defaultFilePath, it appears in
...
-[description],*.extension You can include more than one file type by
-separating the file types with commas.>*Cross-platform note:* The types
-parameter applies only to Windows systems. On Mac OS and Unix systems,
-this parameter has no effect.
+[description],*.extension You can include more than one file type by 
+separating the file types with commas.
+>*Cross-platform note:*  The <types> <parameter> applies only to
+> <Windows|Windows systems>. On <Mac OS> and <Unix|Unix systems>, this
+> <parameter> has no effect.
+
```
Comment (livecodeali) [inline docs/dictionary/command/ask-file.lcdoc]: I think this should be 
```
... 
description, a line feed or comma, and a file extension specification:
    
    [description],*.extension

You can include more than one file type by separating the file types 
with commas.

>*Cross-platform note:* The <types> <parameter> applies only to
...
```

File: docs/dictionary/command/ask-file.lcdoc [hunk 2]
```diff
@@ -42,13 +42,16 @@ suggested file name is provided in the defaultFilePath, it appears in
...
 types:
-The  types parameter specifies which extensions are available at the
+The types parameter specifies which extensions are available at the
 bottom of the dialog box. A file type consists of an optional
 description, a line feed or comma, and a file extension specification:
-[description],*.extension You can include more than one file type by
-separating the file types with commas.>*Cross-platform note:* The types
-parameter applies only to Windows systems. On Mac OS and Unix systems,
-this parameter has no effect.
+  [description],*.extension
```
Comment (montegoulding) [inline docs/dictionary/command/ask-file.lcdoc]: I think you need to add an extra line above and below this one for the code block to work

File: docs/dictionary/command/ask-file.lcdoc [hunk 3]
```diff
@@ -42,13 +42,16 @@ suggested file name is provided in the defaultFilePath, it appears in
...
 bottom of the dialog box. A file type consists of an optional
 description, a line feed or comma, and a file extension specification:
-[description],*.extension You can include more than one file type by
-separating the file types with commas.>*Cross-platform note:* The types
-parameter applies only to Windows systems. On Mac OS and Unix systems,
-this parameter has no effect.
+  [description],*.extension 
+You can include more than one file type by separating the file types 
+with commas.
+>*Cross-platform note:*  The <types> <parameter> applies only to
```
Comment (montegoulding) [inline docs/dictionary/command/ask-file.lcdoc]: Also an extra line between these paragraphs

**Comment by livecodeali** [comment]
@livecode-vulcan review ok e7442f7

**Pull request merged**